python/src/aiconfig/editor/client/src/components/property_controls/UnionPropertyControl.tsx

```diff
@@ -23,7 +24,13 @@ type Props = {
 };
 
 export default memo(function UnionPropertyControl(props: Props) {
-  const { property, renderProperty, setValue, ...renderPropertyProps } = props;
+  const {
+    disabled,
+    property,
+    renderProperty,
+    setValue,
+    ...renderPropertyProps
+  } = props;
```

Comment:
whats the reasoning pass `disabled` as a prop as opposed to useContext -> get readOnly here?

Reply:
Just for consistency with the other "concrete" field renderers (like TextInput, etc.)